src/main/java/seedu/address/logic/PomodoroManager.java

```diff
@@ -142,7 +154,7 @@ public void unpause() throws NullPointerException {
 
     public void reset() {
         timerLabel.textProperty().unbind();
-        timerLabel.setText("POM");
+        pomodoroDisplay.reset();
```

Comment:
Does this mean 25:00 is displayed instead of POM?

Reply:
That's right. "POM" no longer shows. Goes back to the user's specified default timer.